nox/virtualenv.py

```diff
@@ -275,21 +308,26 @@ def bin(self):
             return os.path.join(self.location, "bin")
 
     def create(self):
-        """Create the virtualenv."""
+        """Create the virtualenv or venv."""
         if not self._clean_location():
             logger.debug(
-                "Re-using existing virtualenv at {}.".format(self.location_name)
+                "Re-using existing virtual environment at {}.".format(
+                    self.location_name
+                )
             )
             return False
 
-        cmd = [sys.executable, "-m", "virtualenv", self.location]
-
         if self.interpreter:
-            cmd.extend(["-p", self._resolved_interpreter])
+            cmd = [self._resolved_interpreter]
```

Comment:
This change means that if virtualenv is used, it will now need to be installed *in the resolved interpreter*, whereas the current code only requires virtualenv to be installed in the current interpreter. It's not clear to me whether this would be a problem in practice, but it is a change in behaviour and if it's to be made, it should be done deliberately and not "by accident".

Reply:
Whoops, that was a remnant of my hack to get pipx tests working earlier. I'll add the `-p` flag back in if virtualenv is being used. I'll fix it.